sheet_to_triples/field.py

```diff
@@ -59,7 +59,7 @@ def as_uc(self):
 
     @property
     def as_text(self):
-        return _must(self._value.strip())
+        return _must(str(self).strip())
```

Comment:
Not completely sure about this, as it feels like we need a way of rejecting a column full of numbers or dates if we're expecting text, and the python stringification is sometimes just the wrong thing, eg for `None` from an empty cell.

Reply:
So I kind of dealt with this problem with the `date_or_text` hack -- two implementations of the method, the strict one which falls over if the source data is the wrong format, and the flabby one which wraps a `try/except` around the strict one and does something else if it fails.  But then writing two methods for every cell value conversion doesn't exactly scale.

Ultimately I feel that this is not a problem that we can apply strict type requirements to -- or at least, not a problem solved by applying strict type requirements to. N consultants making spreadsheets are going to find N different ways of structuring the data, and then Excel jumps in to helpfully mangle it some more, and then you've got human error involved where somebody makes a typo or misformats a cell and the resulting bad value makes ingest of a 6,000 row spreadsheet fall over completely. Either the process for finding and fixing bad values in the source data needs to be much better than it currently is (i.e. practically non-existent) or we're going to have to be somewhat generous with how we interpret it during ingest.

Reply:
Although an interim solution could be to restructure to reject bad values without the entire thing blowing up.